<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWFormSetWeight2</class>
 <widget class="QWidget" name="QWFormSetWeight2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>278</width>
     <height>244</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="overall">
    <attribute name="title">
     <string>Overall</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="tree_overall">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <column>
        <property name="text">
         <string>Test Conditions</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Weight_F (%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Weight_R (%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Comments</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="general">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>General</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="tree_general">
       <column>
        <property name="text">
         <string>Parameters</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Weight_F (%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Weight_R (%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Comments</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="longi">
    <attribute name="title">
     <string>Longitudinal</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="tree_longi">
       <column>
        <property name="text">
         <string>Parameters</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Weight_F (%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Weight_R (%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Comments</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="lateral">
    <attribute name="title">
     <string>Lateral</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="tree_lateral">
       <column>
        <property name="text">
         <string>Parameters</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Weight_F (%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Weight_R (%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Comments</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="vertical">
    <attribute name="title">
     <string>Vertical</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="tree_vertical">
       <column>
        <property name="text">
         <string>Parameters</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Weight_F (%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Weight_R (%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Comments</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="steer">
    <attribute name="title">
     <string>Steer</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="tree_steer">
       <column>
        <property name="text">
         <string>Parameters</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Weight_F (%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Weight_R (%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Comments</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="roll">
    <attribute name="title">
     <string>Roll</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="1" column="0">
      <widget class="QTreeWidget" name="tree_roll">
       <column>
        <property name="text">
         <string>Parameters</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Weight_F (%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Weight_R (%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Comments</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>185</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Front Suspension (%):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>9</y>
     <width>176</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rear Suspension (%):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="weightF">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>9</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border: 2px groove gray;
	border-radius: 14px;
	padding: 2px 4px;
	color: rgb(0, 0, 0);
	border-color: rgb(255, 140 ,0);
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="weightR">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>9</y>
     <width>20</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border: 2px groove gray;
	border-radius: 14px;
	padding: 2px 4px;
	color: rgb(0, 0, 0);
	border-color: rgb(255, 140 ,0);
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <action name="actImport">
   <property name="text">
    <string>Import Setting</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>Save Setting As</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actResetAll">
   <property name="text">
    <string>Reset All</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actResetLoc">
   <property name="text">
    <string>Reset Current Page</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actSeparator1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </action>
  <action name="actSeparator2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </action>
  <action name="actSeparator3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
